site/src/sphinx/setup-configuration.rst

```diff
@@ -48,11 +48,15 @@ defaults:
       "maxNumFilesPerMirror": null,
       "maxNumBytesPerMirror": null,
       "writeQuotaPerRepository": {
-        "requestQuota" : 5,
+        "requestQuota": 5,
         "timeWindowSeconds": 1
       },
       "accessLogFormat": "common",
-      "authentication": null
+      "authentication": null,
+      "cors": {
+         "allowedOrigins": ["http://example1.com", "http://example2.com"],
+         "maxAge": 1800
+      }
```

Comment:
CORS is an optional feature that users should explicitly configure to use.
Some users copy the configuration without deep thought. For that reason, I'd like to make it null here. 
```suggestion
      "cors": null
```

Reply:
i agree